package.json

```diff
@@ -216,6 +215,7 @@
     "nodemon": "^2.0.22",
     "npm-run-all": "^4.1.5",
     "prop-types": "^15.8.1",
+    "react-scripts": "5.0.1",
```

Comment:
For my understanding, what's the significance of moving react-scripts to a dev dependency in this PR?

Reply:
Follow-up question: does the scan only apply to `dependencies` and not to `devDependencies`?

Reply:
> Follow-up question: does the scan only apply to dependencies and not to devDependencies?

Yep, it skips `devDependencies` https://github.com/aquasecurity/trivy/discussions/3714

I don't think `react-scripts` needs to be in `dependencies` since it might be a dev-only dependency, and since it has the vulnerability that failed the build previously, it should be safe to move it over to `devDependencies`

@ssundahlTTD can confirm

Reply:
excellent, thanks for explaining!

Reply:
There's definitely [discussion](https://github.com/facebook/create-react-app/issues/2696) as to whether `react-scripts` should be installed as a `devDependecy` by default by create-react-app, but in any case it looks safe to move 👍